elfi/client.py

```diff
@@ -70,37 +71,79 @@ def total(self):
 
     @property
     def num_ready(self):
-        return self.total - len(self.pending_indices)
+        return self.total - self.num_pending
```

Comment:
I'm so used to `n_whatever` for numbering that I had to think a while if you meant something else. :) But not a big thing, of course, so as you wish.